src/components/v5/common/ActionSidebar/partials/forms/PaymentBuilderForm/partials/PaymentBuilderRecipientsField/hooks.tsx

```diff
@@ -119,9 +113,11 @@ export const useRecipientsFieldTableColumns = (
                 id: 'totalDelay',
                 header: () => formatText({ id: 'table.column.totalDelay' }),
                 cell: ({ row }) => {
-                  const totalHours =
-                    expendituresGlobalClaimDelayHours +
-                    (dataRef.current[row.index]?.delay || 0);
+                  const totalHours = BigNumber.from(
+                    expendituresGlobalClaimDelayHours,
+                  )
+                    .add(BigNumber.from(dataRef.current[row.index]?.delay || 0))
+                    .toNumber();
```

Comment:
This can't be easily tested since there's no saga to set the global claim delay, however I spotted some issues:

The problem with `toNumber` here is that it's possible for `expendituresGlobalClaimDelayHours` to be > 2^53 which will cause it to throw an error (and it wouldn't be caught). All claim delay calculations should use BigNumbers (great shout on already switching it here) and strings.

This made me realise the `expendituresGlobalClaimDelayHours` will not be set correctly because of line 29 where we check if global claim delay's type is `number`, which it will never be.

And, if `dataRef.current[row.index]?.delay` is a floating point number, `BigNumber` will throw an error. 

Feel free to ping me on Discord and I'll provide you with instructions on how to set the global claim delay for testing.